inst/WORDLIST

```diff
@@ -245,6 +259,7 @@ microsoft
 mkdir
 modelled
 mS
+mutantdino
```

Comment:
sounds scary!

Reply:
ha its a usename for one of the peeps writing vscode plug ins. the lintr spell checker just likes looking everywhere.